### PR TITLE
Simplify and ES6-ify

### PR DESCRIPTION
**Pull request by davidsawyer**
This PR includes a handful of small tweaks and optimizations, leveraging ES6 wherever possible.

The two largest changes are:
- Instead of passing around `null` to our different functions, we can just throw the error on the spot, `catch` the error, and then log the error. Let me know what you think about logging the error vs. not logging the error. I don't have a strong opinion either way.
- Removing the `setInterval` code. Chrome is very helpful in that it will inject the extension's code right when the DOM is loaded, so we shouldn't need to listen to `document.readyState` ourselves.

### CODE REVIEW

File: src/inject.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /* global chrome, fetch*/
```
Comment (harshjv) [inline src/inject.js]: standard-linter: remove chrome from global

File: src/inject.js [hunk 2]
```diff
@@ -19,9 +19,9 @@ function formatKiloBytes (bytes) {
 
   bytes *= 1024
 
-  var K = 1024
-  var MEASURE = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB']
-  var i = Math.floor(Math.log(bytes) / Math.log(K))
+  const K = 1024,
+    MEASURE = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'],
```
Comment (harshjv) [inline src/inject.js]: standard-linter: split const declaration into multiple statements

File: src/inject.js [hunk 3]
```diff
@@ -30,55 +30,47 @@ function formatKiloBytes (bytes) {
 }
 
 function checkStatus (response) {
-  if (response.status >= 200 && response.status < 300) {
```
Comment (harshjv) [inline src/inject.js]: standard-linter: expected literal on the right side of <=

**Comment by harshjv** [comment]
Hey @davidsawyer 

Thanks for the PR!

Can you take a look at the comments? We follow [feross/standard](https://github.com/feross/standard)

I'll add Travis CI to automate the process. #8

**Comment by davidsawyer** [comment]
sounds good 👍 

just took care of those.

**Pull request merged**